chrome/chrome.d.ts

```diff
@@ -2422,8 +2422,8 @@ declare module chrome.webRequest {
     }
 
     interface WebRequestAuthRequiredEvent extends chrome.events.Event { 
-        addListener(callback: (details: OnAuthRequiredDetails, callback?: (response: BlockingResponse) => void, filter?: RequestFilter, opt_extraInfoSpec?: string[]) => void): void;
-        removeListener(callback: (details: OnAuthRequiredDetails, callback?: (response: BlockingResponse) => void) => void): void;
+        addListener(callback: (details: OnAuthRequiredDetails) => BlockingResponse, filter?: RequestFilter, opt_extraInfoSpec?: string[]): void;
+        removeListener(callback: (details: OnAuthRequiredDetails) => BlockingResponse) : void;
```

Comment:
You are correct.
I wrote the correct callback type now.
The bug was that filter and opt_extraInfoSpec where part of callback and not part of addListener.

According to this: https://developer.chrome.com/extensions/webRequest#event-onAuthRequired